core/operation/src/main/java/uk/gov/gchq/gaffer/operation/util/OperationConstants.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2017 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.gaffer.operation.util;
+
+public final class OperationConstants {
```

Comment:
Would it be better for these to be public constants inside the While operation class? Do we gain anything from having an additional utility class just for these constants?